models/engine/db_storage.py

```diff
@@ -51,6 +51,14 @@ def all(self, cls=None):
                     new_dict[key] = obj
         return (new_dict)
 
+    def get(self, cls, id):
+        """retrieve one object."""
+        return self.all(cls).get(cls + '.' + id)
+
+    def count(self, cls=None):
+        """returns number of objects in storage"""
+        return len(self.all(cls))
```

Comment:
Very concise!  Since these methods are identical in FileStorage consider putting them on a base storage class that both storage engines inherit from!